player/lua/osc.lua

```diff
@@ -2984,6 +2989,9 @@ end
 opt.read_options(user_opts, "osc", function(changed)
     validate_user_opts()
     set_osc_styles()
+    if changed.timetotal or changed.timems then
```

Comment:
Sorry to be nitpicky, but this time I would be explicit and check them separately.